### PR TITLE
deployment script updates

### PR DESCRIPTION
**Pull request by stuartwk**
updates deployment scripts to use SwapAndStakePolygon and SwapAndStakeL1

### CODE REVIEW

File: scripts/deploy-swap-v2-l1-mainnet.ts [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import { ethers } from 'hardhat'
+import { SwapAndStakeV2L1 } from '../typechain'
+
+async function main() {
+	const uniswapRouterAddress = '0x7a250d5630B4cF539739dF2C5dAcb4c659F2488D'
+	const devAddress = '0x5caf454ba92e6f2c929df14667ee360ed9fd5b26'
+	const lockupAddress = '' // What should this be?
```
Comment (stuartwk) [inline scripts/deploy-swap-v2-l1-mainnet.ts:7]: @aggre what should this be? Should we be able to dynamically set this in the contract?

Comment (aggre) [reply scripts/deploy-swap-v2-l1-mainnet.ts:7]: Yes, the address of Lockup on L1 should be used the rerurn value of `IAddressConfig(address).lockup()`

Comment (aggre) [reply scripts/deploy-swap-v2-l1-mainnet.ts:7]: But, I think we can deploy them for L2 first

**Review by aggre — APPROVED** [review]
I'll merge this, and let's add a fixing for https://github.com/dev-protocol/eth-swap-and-stake/pull/19/files#r921943150 later

**Pull request merged**